src/styles/inputs.scss

```diff
@@ -41,14 +41,14 @@ $focusedLabelTop: 5px;
 
   &--empty {
     &:not(#{$f}--active):after {
-      background-image: url('~hd-blocks/assets/icons/ic-error.svg');
+      background-image: url('/assets/icons/ic-error.svg');
```

Comment:
This and all the the paths in this file won't work.
`/assets/icon.svg` with point to `PATH/assets/icon.svg` (`PATH == "blocks.homeday.de"` in the case of storybook for example), which is not what we want, we should use a relative path to the file locally.
The issue we will face here if we use `../assets/` like in the other case, is that this file is imported in other components, which means that the path will be resolved based on the location of the component where it's imported.
```
/assets
/directory1
  |_Component1.vue
  |_/directory2
     |_Component2.vue
```
So in the example above the icon will work on `Component1` but not on `Component2`.

Reply:
The icon works for both components because I'm not using relative paths inside `inputs.scss` so once imported it won't be relative.

It is indeed a limitation from Sass in which it doesn't respect the relative paths from the imported component.

The `PATH` will be the root path used for the project in which for blocks it will be `blocks.homeday.de` but for every other project it will follow its root path.

Nevertheless it will work as expected as it will be built by the "father" project which will handle those files during build.

Example of a built project using this new structure:
![image](https://user-images.githubusercontent.com/1734873/73350805-03841b00-428e-11ea-9a0a-39ffb9c9b721.png)
![image](https://user-images.githubusercontent.com/1734873/73350833-0ed74680-428e-11ea-8769-9e9f91e4d178.png)

To be honest I don't like this solution that much as it is kind of "magical". There is this guy: https://github.com/bholloway/resolve-url-loader but it will be necessary to add it to the "father" project anyway.

@homeday-de/frontend-engineering thoughts here?

Reply:
`ic_visibility-on.svg` for example will work because it's defined as `../../assets/icons/ic_visibility-on.svg` inside HdInputPassword, but the example above (`ic-error.svg`) won't work, the `/assets/icons/ic-error.svg` won't be touched by webpack because it's not a path, and the browser will try to access it as a URL, like:
`YOUR_APP_BASE/assets/icons/ic-error.svg` which doesn't exist, instead we want something similar to: `YOUR_APP_BASE/img/ic-error.HASH.svg`

Reply:
@ilyasmez as we've just aligned I'm going to use `homeday-blocks` as an alias so we can resolve it properly thanks to https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules

Gonna rework on it soon!